app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Avatar, Button, Table } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACTS_LIST, COLUMN_DATA } from "./constants";
+
+const Contacts = () => {
+  const [currentPageNumber, setCurrentPageNumber] = useState(1);
+  // const [showNewContactPane, setShowNewContactPane] = useState(false);
```

Comment:
I didnt remove this line of code as I just have to uncomment it for the working of New contact pane.

From Now on, I will not keep in my about the new functionalities while working on one as it is leading to commented code. Will take one functionality after other.

Reply:
That's not the right way to work on a project that is maintained using `git`. When using git you can create different branches that deal with different features. If you feel that a particular file or code doesn't belong to the current branch, then rather than commenting it out, stash it and apply it the appropriate branch.